src/components/PledgeCard.js

```diff
@@ -35,19 +35,19 @@ function PledgeCard({ value, real }: Props): Node {
   const realValue = Number(value) / 1000000; // divided in 1,000,000 to convert from Lovelace to ADA
 
   switch(true){
-  case realValue >= 1e6:
+  case realValue >= 250000:
     backgroundColor = '#BFC8FF';
     break;
-  case realValue < 1e6 && realValue >= 150e3:
+  case realValue < 200000 && realValue >= 100000:
     backgroundColor = '#D6DDF5';
     break;
```

Comment:
@cardanians , I can see one problem in the new ranges that there's a gap between 200k and 250k where the background color value would not be applied, because the checks are:

1. If >= 250K
2. ELSE if < 200k and >= 100K

Reply:
I think it might be better to use if-else there with no upper limit checks, so we only gonna need to check if the pledge is `>=100k` or `>=250k`